sdk/tests/conformance2/transform_feedback/simultaneous_binding.html

```diff
@@ -125,8 +132,14 @@
 gl.uniformBlockBinding(prog, ubi, 0);
 gl.bindBufferBase(gl.UNIFORM_BUFFER, 0, uniformBuffer);
 
-drawWithFeedbackBound(gl, prog, vao, tf, true);
+let drawArrays = ()=>gl.drawArrays(gl.POINTS, 0, 4);
+let drawElements = ()=>gl.drawElements(gl.POINTS, 4, gl.UNSIGNED_SHORT, 0);
```

Comment:
What about draw*Instanced?

Reply:
Done. In the process I discovered some problems with draw*Instanced buffer size checking. New test here: https://github.com/KhronosGroup/WebGL/pull/2604